lib/membrane/video_compositor/object/layout.ex

```diff
@@ -58,9 +58,9 @@ defmodule Membrane.VideoCompositor.Object.Layout do
   defmodule RustlerFriendly do
     @moduledoc false
     # A rustler-friendly version of the Layout, prepared for rust serialization
-    alias Membrane.VideoCompositor.Scene.Object.Layout
-    alias Membrane.VideoCompositor.Scene.Object.RustlerFriendly, as: RFObject
-    alias Membrane.VideoCompositor.Scene.Resolution
+    alias Membrane.VideoCompositor.Object.Layout
+    alias Membrane.VideoCompositor.Object.RustlerFriendly, as: RFObject
+    alias Membrane.VideoCompositor.Resolution
```

Comment:
`Object.RustlerFriendly` is on the same level as `Object.Layout` and I am not the biggest fan of it, but I guess there is no way to mitigate it with our approach. In case you can think of some solution, I'm leaving it for a reconsideration :D